### PR TITLE
Properly label PRs from Scala Steward

### PR DESCRIPTION
**Pull request by sideeffffect**
For some reason I forgot to add this :facepalm:

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
     conditions:
       - author=scala-steward
-      - base=master
-      - status-success=ci
-      - "body~=(labels: library-update, semver-minor)|(labels: library-update, semver-patch)|(labels: sbt-plugin-update, semver-minor)|(labels: sbt-plugin-update, semver-patch)|(labels: scalafix-rule-update)|(labels: test-library-update)"
     actions:
-      merge:
-        method: merge
+      assign:
+        users: [eikek]
```
Comment (eikek) [inline .mergify.yml:7]: Is this necessary for some workflow? it assigns myself to all scala-steward prs?

File: .mergify.yml [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+      label:
+        add: ["type: dependencies"]
   - name: label scala-steward's breaking PRs
     conditions:
       - author=scala-steward
       - "body~=(labels: library-update, semver-major)|(labels: sbt-plugin-update, semver-major)"
     actions:
       label:
         add: ["type: breaking"]
```
Comment (eikek) [inline .mergify.yml:16]: That's really nice! Haven't looked too deep into mergify docs yet :-)

File: .mergify.yml (hunk 1 above)
Comment (sideeffffect) [reply .mergify.yml:7]: Not really, if you don't want to. But some PR might not get merged (for whatever reason), so you may want to be notified of it :wink:

**Comment by eikek** [comment]
The funny thing is that this one triggered the bug #184 on travis :-)

**Pull request merged**